src/modules/account/containers/AccountDetails/SurplusCard.tsx

```diff
@@ -21,27 +23,135 @@ export function SurplusCard() {
+        margin: auto 0 0;
+      }
+    }
+
+    ${InfoCard} > div > span {
+      display: flex;
+      flex-flow: column wrap;
+      align-items: center;
+      justify-content: center;
+    }
+
+    ${InfoCard} > div > span > i,
+    ${InfoCard} > div > a,
+    ${InfoCard} > div > span > p {
+      display: flex;
+      font-size: 13px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 1.1;
+      width: 100%;
+      text-align: center;
+      justify-content: center;
+      align-items: center;
+      color: ${({ theme }) => transparentize(0.3, theme.text1)};
+    }
+
+    ${InfoCard} > div > span > p {
+      color: ${({ theme }) => theme.text1};
+    }
+
+    ${InfoCard} > div > span > b {
+      font-size: 28px;
+      font-weight: bold;
+      color: ${({ theme }) => theme.success};
+      width: 100%;
+      text-align: center;
+      margin: 20px auto 0;
+      word-break: break-all;
+    }
+
+    ${InfoCard} > div > a {
+      margin: 20px auto 0;
+    }
+
+    ${InfoCard} > div > small {
+      font-size: 15px;
+      font-weight: 500;
+      line-height: 1.1;
+      color: ${({ theme }) => transparentize(0.5, theme.text1)};
+      margin: 3px auto 0;
+    }
+
+    ${QuestionWrapper} {
+      opacity: 0.5;
+      transition: opacity 0.2s ease-in-out;
+
+      &:hover {
+        opacity: 1;
+      }
+    }
+  `
+
   return (
-    <InfoCard>
-      <span>
-        Your total surplus{' '}
-        <MouseoverTooltipContent content={'TODO: insert tooltip'} wrap>
-          <HelpCircle size={14} />
-        </MouseoverTooltipContent>
-      </span>
-      <span>
-        {isLoading
-          ? 'Loading...'
-          : surplusAmount && (
-              <>
-                +<TokenAmount amount={surplusAmount} tokenSymbol={surplusAmount?.currency} />
-              </>
-            )}
-        {!surplusAmount && 'No surplus for the given time period'}
-      </span>
-      {surplusUsdAmount && <FiatAmount amount={surplusUsdAmount} accurate={false} />}
-      {/* TODO: add correct link */}
-      <ExternalLink href={'https://swap.cow.fi'}>Learn about surplus on CoW Swap ↗</ExternalLink>
-    </InfoCard>
+    <Wrapper>
+      <InfoCard>
+        <div>
+          <span>
+            <i>Your total surplus <QuestionHelper text={'TODO: insert tooltip'} /></i>
+          </span>
+          <span>
+            {isLoading
+              ? <p>Loading...</p>
+              : surplusAmount && (
+                  <b>
+                    +<TokenAmount amount={surplusAmount} tokenSymbol={surplusAmount?.currency} />
+                  </b>
+                )}
+            {!surplusAmount && <p>No surplus for the given time period</p>}
+          </span>
+          <small>{surplusUsdAmount && <FiatAmount amount={surplusUsdAmount} accurate={false} />}</small>
+        </div>
+        <div>
+          {/* TODO: add correct link */}
+          <ExternalLink href={'https://swap.cow.fi'}>Learn about surplus on CoW Swap ↗</ExternalLink>
```

Comment:
We got a link for this: 

```suggestion
          <ExternalLink href={'https://blog.cow.fi/announcing-cow-swap-surplus-notifications-f679c77702ea'}>Learn about surplus on CoW Swap ↗</ExternalLink>
```

From https://cowservices.slack.com/archives/C05E2UZCJ3F/p1688373479915889